server/chat/backend/agent/tools/cloud_tools.py

```
@@ -1730,6 +1744,42 @@ def _pinned_trigger(action_id: str = "", _pid=pinned_id, _fn=final_func, **kw):
+        final_os_indices = (
+            wrap_func_with_capture(notification_wrapped_os_indices, "list_opensearch_indices")
+            if tool_capture else notification_wrapped_os_indices
+        )
+        tools.append(StructuredTool.from_function(
+            func=final_os_indices,
+            name="list_opensearch_indices",
+            description="List available OpenSearch indices to discover what log data is available for searching.",
+            args_schema=OpenSearchListIndicesArgs,
+        ))
+
+        logging.info(f"Added 2 OpenSearch tools for user {user_id}")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)